powerpacks.go

```diff
@@ -300,6 +372,18 @@ func (service *PowerpackService) Find_shortcode(shortcode string) (response *Fin
 	return
 }
 
+func (service *PowerpackService) Find_tollfree(tollfree string) (response *FindTollfreeResponse, err error) {
+	numberpool_path := service.Powerpack.NumberPoolUUID
```

Comment:
it should be numberpoolUUID right, not numberpool_path

Reply:
it was named numberpool_path in existing apis so i kept the same name